Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Params
+ARG RAILS_ENV="production"
+
+# Defining ruby version
+FROM ruby:2.6.6
+
+# Set working dir and copy app
+WORKDIR /usr/src/app
+COPY . .
+
+# Prerequisites for gems install
+RUN apt-get install tzdata \
+                    git
+
+# Install bundler and gems
+RUN gem install bundler
```

Comment:
Similarly I would parametarise the bundler version. Add `ARG BUNDLER_VERSION=2.2.16` (use an appropriate default) to the top of the file and then rewrite this line as `RUN gem install bundler:$BUNDLER_VERSION`